test/SparkLendMainnetIntegration.t.sol

```diff
@@ -158,32 +144,7 @@ contract SparkLendMainnetIntegrationTest is Test {
         _triggerUpdate(ETH);
 
         // slope1 has adjusted down a bit so the borrow rate is slightly lower at same utilization
-        assertEq(_getBorrowRate(ETH), 0.027800738894650680218981526e27);
-    }
-
-    function test_usdc_usdt_market_oracles() public {
```

Comment:
Why was this test removed?

Reply:
I had to bump the block, and we've already applied these upgrades to the market.

Reply:
I could re-add, but not sure if it provides much value.

Reply:
I think its a good idea to keep these tests and set the block within the test

Reply:
ok re-added